### PR TITLE
Make CreateEmpty implement KeywordExecutable

### LINKED ISSUE #2360: Make CreateEmpty server utility KeywordExecutable
The CreateEmpty server utility could have a KeywordExecutable, like other server utilities, to make it easier to run.

### PR DESCRIPTION
**Pull request by dlmarion**
Closes #2360

### CODE REVIEW

File: core/src/main/java/org/apache/accumulo/core/file/rfile/CreateEmpty.java [hunk 1]
```diff
@@ -40,7 +41,7 @@
  * Create an empty RFile for use in recovering from data loss where Accumulo still refers internally
  * to a path.
  */
-public class CreateEmpty {
+public class CreateEmpty implements KeywordExecutable {
```
Comment (DomGarguilo) [inline core/src/main/java/org/apache/accumulo/core/file/rfile/CreateEmpty.java:44]:
```suggestion
@AutoService(KeywordExecutable.class)
public class CreateEmpty implements KeywordExecutable {
```
Should this be added here?

Comment (dlmarion) [reply core/src/main/java/org/apache/accumulo/core/file/rfile/CreateEmpty.java:46]: Yep, thanks. I forgot the annotation. I updated both PRs.

**Comment by milleruntime** [comment]
This one will probably need to be added to KeywordStartIT as well.

File: core/src/main/java/org/apache/accumulo/core/file/rfile/CreateEmpty.java [hunk 2]
```diff
@@ -76,10 +79,25 @@ public void validate(String name, String value) throws ParameterException {
...
+  }
+
+  @Override
+  public String keyword() {
+    return "create-empty";
+  }
+
+  @Override
+  public String description() {
+    return "creates an empty rfile";
```
Comment (DomGarguilo) [inline core/src/main/java/org/apache/accumulo/core/file/rfile/CreateEmpty.java:92]:
```suggestion
    return "Creates an empty rfile";
```
The rest of the descriptions start with a capital letter.

**Review by DomGarguilo — APPROVED** [review]

**Pull request merged**